apps/hash-frontend/src/components/hooks/use-account-pages.ts

```diff
@@ -16,7 +16,7 @@ export type AccountPagesInfo = {
 };
 
 export const useAccountPages = (
-  ownedById: OwnedById,
+  ownedById?: OwnedById,
```

Comment:
We could amend the `skip` line below so that the query isn't even attempted if this is called without an `ownedById`
```ts
    skip: !ownedById || !hashInstance?.properties.pagesAreEnabled,
```